### DISCUSSION TITLE
Does multi-point optimization apply to unsteady states?

### DISCUSSION THREAD

**Discussion by wndzjl** [mdolab/dafoam#720] [General]
I would like to ask if dafoam can do unsteady multi-point optimization. In the future, I want to do a combination of unsteady and steady optimization design, is it feasible?

**Comment by friedenhe** [comment]
Not yet. This is something we are working on. It will probably be ready in early 2025.

**Reply by wndzjl** [reply]
I am also working on this, it is currently operational, still in the testing phase. In the future, I may try to combine both steady and unsteady state multi-point optimization. That is, in the optimization process, different optimizers are matched for different attack angles. Please let me know if you release a unsteady multi-point optimization version, thank you very much！

**Comment by friedenhe** [comment]
Please follow DAFoam's update. This capability will be added in version 4.

**Reply by wndzjl** [reply]
According to your other response, can I refer to the code in `multicase` for unsteady multi-point optimization. I don't quite understand the difference between `multicase` and `multipoint`. Is it true that all cases that can be done by `multipoint` can be done by `multicase`? But the cases that can be done with `multicase` may not necessarily be able to be done with `multipoint`.

**Reply by friedenhe** [reply]
> According to your other response, can I refer to the code in `multicase` for unsteady multi-point optimization. I don't quite understand the difference between `multicase` and `multipoint`. Is it true that all cases that can be done by `multipoint` can be done by `multicase`? But the cases that can be done with `multicase` may not necessarily be able to be done with `multipoint`.

That is right.
